package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@gnosis.pm/gp-v1-ui",
+  "name": "@gnosis.pm/gp-v1",
```

Comment:
Well, the package name is tricky.
It contains gp-v1-ui, yes, but also the explorer and the trade ui.

Should it be named something else? Or just `gp-ui` ?

Reply:
`gp-ui` sounds good to me

Reply:
of course! this is me being dummmb. I wanted to drop the v1, not the ui!